src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -224,12 +224,20 @@
 
 (defn f-view-internal
   [{:keys [theme]}]
-  (let [featured-communities (rf/sub [:communities/featured-contract-communities])]
+  (let [featured-communities (rf/sub [:communities/featured-contract-communities])
+        customization-color  (rf/sub [:profile/customization-color])]
     [rn/view
      {:style (style/discover-screen-container (colors/theme-colors
                                                colors/white
-                                               colors/neutral-95))}
-     [discover-screen-content featured-communities theme]]))
+                                               colors/neutral-95
+                                               theme))}
+     [discover-screen-content featured-communities theme]
+     [quo/floating-shell-button
+      {:jump-to {:on-press            #(rf/dispatch [:shell/navigate-to-jump-to])
+                 :customization-color customization-color
+                 :label               (i18n/label :t/jump-to)}}
+      {:position :absolute
```

Comment:
could this and every other inline style be moved to a styles file?

Reply:
done, thank you